JunStack.java

```diff
@@ -1,3 +1,5 @@
+import java.util.Arrays;
+
 public class JunStack {

```

Comment:
Great job on this class as a whole, each method was done correctly and linted by the end of it, which has helped my sleep at night. The code was simple and readable, the best type.

Reply:
love this